src/features/Home/GetStarted/GetStarted.module.scss

```diff
@@ -0,0 +1,82 @@
+.mainPageRow {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: fit-content;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  padding: 80px 0;
+
+  .columnContainer {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+
+    .cardContainer {
+      display: flex;
+      flex-direction: row;
+      margin-bottom: 40px;
+      @media screen and (max-width: 992px) {
+        flex-direction: column;
+        gap: 2rem;
+      }
+    }
+    .mainPageCard {
+      width: 100%;
+      max-width: 316px;
+      padding: 32px 24px;
+      border-radius: 4px;
+      box-shadow: 0 0.4rem 0.8rem 0 rgb(14, 14, 14, 0.1);
+      background-color: var(--ifm-color-white);
+      margin: 0 12px;
+      font-size: 16px;
+      line-height: 1.5;
+      color: var(--ifm-color-black);
+      margin-bottom: 0;
+
+      .header {
+        display: flex;
+        width: 100%;
+        justify-content: space-between;
+        .guideIcon,
+        .signUpIcon,
+        .registerYourAppIcon {
+          width: 32px;
+          height: 32px;
+          background-repeat: no-repeat;
+          background-position: center;
+        }
+      }
+    }
+  }
+}
+
+.guideIcon,
+.signUpIcon,
+.registerYourAppIcon {
+  width: 3.2rem;
+  height: 3.2rem;
```

Comment:
Also convert these rem values to the rem function values so that we use the correct sizes. So in this case `width: rem(3.2);` for example.